### DISCUSSION TITLE
Draw和Measure组件如何默认收起，autoExpand属性不起作用

### DISCUSSION THREAD

**Discussion by ChauPha** [zouyaoji/vue-cesium#175] [Q&A] [answered]
Draw和Measure组件如何默认收起，autoExpand属性不起作用
![1648802449(1)](https://user-images.githubusercontent.com/24471468/161227998-beee80b0-e1cc-40fa-b426-19ef19424fbe.jpg)

**Comment by zouyaoji** [comment, marked as answer]
支持的哈，但最新版本已经不用这个属性了。用 modelValue 属性。mainFabOpts 里面哈。

**Reply by ChauPha** [reply]
好的 多谢哈 对于draw功能我有一系列的需求想请教下 方便解答吗

**Reply by zouyaoji** [reply]
有啥不方便的啊？

**Reply by ChauPha** [reply]
我这边打印出绘制选项参数与官方文档的不一样呢 少了些属性 是我版本问题吗
![1648871173(1)](https://user-images.githubusercontent.com/24471468/161369165-423e821a-1f00-4245-907d-cf0544b1a361.jpg)
![1648871140(1)](https://user-images.githubusercontent.com/24471468/161369169-f2947370-8f1c-4e25-8bd7-a7b7f09cc7f8.jpg)

**Reply by zouyaoji** [reply]
是哈，这些都是后来完善的。

**Comment by ChauPha** [comment]
首次改变单个标记属性（polygon的填充色）可以生效，但是之后再改变的话，界面不会自动渲染呢
![1648890142(1)](https://user-images.githubusercontent.com/24471468/161376530-35337c3e-3f93-4179-98a9-2fd3f67da180.jpg)
@zouyaoji 大佬帮我看看

**Reply by zouyaoji** [reply]
像我这样改试试。
![录制_2022_04_02_20_41_57_181](https://user-images.githubusercontent.com/24430116/161383858-dbd7d666-f170-41c4-b8bb-41980d5f37cd.gif)

**Reply by ChauPha** [reply]
我在修改属性的时候 大部分是可以响应的 但是polylineOpts.width和pointOpts.pixelSize响应有点问题  然而我在官网上测试是可以的  问题是出在哪了呢
![1649229646(1)](https://user-images.githubusercontent.com/24471468/161918220-dc80cde5-8758-433f-a8f6-7444b021c6b9.jpg)
